### PR TITLE
Add autoupdate config in froala-editor 

### PR DESCRIPTION
**Pull request by stefanneculai**
For #6307.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Sorry @stefanneculai , would you please mention the lib name in the first line of your commit message for that we can easily trace the changes from commit log? Thanks.

**Comment by stefanneculai** [comment]
@PeterDaveHello Done.

**Comment by PeterDaveHello** [comment]
@stefanneculai do you need files under `plugins`, `themes` in the css folder?
If so, we need to update the auto-update config.

**Comment by stefanneculai** [comment]
Yes. Should `js/plugins/*` and so on be added?

**Comment by PeterDaveHello** [comment]
if so, you should use `css/**/*` instead of `css/*` to grab files recursively, but in fact we usually don't use this kind of wildcard under a non-dist folder or repository, do you have any suggestions? but if you still wanna use that, I'll accept your PR as you are the author of the lib.

**Comment by stefanneculai** [comment]
I have made the update. Actually we use the whole repository as a dist folder. We are doing the development of the library only internal and on Github we make public only the dist of the package.

File: ajax/libs/froala-editor/package.json [hunk 1]
```diff
@@ -1,59 +1,38 @@
...
+      "type": "Advanced",
+      "url": "https://www.froala.com/wysiwyg-editor/pricing"
     },
     {
       "type": "OEM",
-      "url": "http://editor.froala.com/download"
+      "url": "https://www.froala.com/wysiwyg-editor/pricing"
     }
-  ],
-  "repository": {
```
Comment (PeterDaveHello) [inline ajax/libs/froala-editor/package.json]: Can we reserve repository info?

Comment (stefanneculai) [inline ajax/libs/froala-editor/package.json]: Added it back.

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**